splunk-quarkus/src/main/java/com/redhat/console/notifications/splunkintegration/SplunkIntegration.java

```diff
@@ -129,16 +129,26 @@ private void configureHandler() throws Exception {
             // Transform message to add splunk wrapper to the json
             .transform().simple("{\"source\": \"eventing\", \"sourcetype\": \"Insights event\", \"event\": ${body}}")
 
+            // Redirect depending on http or https (different default ports) so that it goes to the default splunk port
             // Send the message to Splunk's HEC as a splunk formattted event.
             // It sends token via Basic Preemptive Authentication.
             // POST method is being used, set up explicitly
             // (see https://camel.apache.org/components/latest/http-component.html#_which_http_method_will_be_used).
-            .toD(http("$simple{headers.metadata[url]}/services/collector/event")
-                .authenticationPreemptive(true)
-                .authMethod("Basic")
-                .httpMethod("POST")
-                .authUsername("x")
-                .authPassword("$simple{headers.extras[token]}"))
+            .choice()
+                .when().simple("${headers.metadata[url]}" startsWith "https")
+                    .toD(http("https://$simple{headers.metadata[url][hostname]}:8088/services/collector/event")
```

Comment:
If the `headers.metadata[url]` is a string how the `[hostname]` would take just a hostname out of it?

Reply:
@CodeHeeler, you can try to use `headers.metadata[url]` as is for the `http()` (and `https()` call). We can require the port to be filled in by customer. The added path (`/services/collector/event`) part can stay as is.